recipe-server/normandy/settings.py

```diff
@@ -380,6 +385,9 @@ class Development(Base):
     SWAGGER_SETTINGS = Base.SWAGGER_SETTINGS
     SWAGGER_SETTINGS['VALIDATOR_URL'] = None
 
+    # URL that hosts Experimenter experiment definitions
+    EXPERIMENTER_PENDING_URL = values.URLValue('https://localhost/api/v1/experiments/?status=Pending')
```

Comment:
It probably makes sense either to add Experimenter to the docker-compose setup, or at least add a reference to the developer docs about how to set up an instance.